index.html

```diff
@@ -174,56 +289,89 @@ <h4 class="about-l">About me</h4>
       <button type="submit" class="see-proj">Get My Resume</button>
     </div>
     <hr class="hr"/>
-
-    <div class="skills">
+<div class="skills">
        
-  <ul>
+<ul class="skills-display">
+  
 
       <li class="main">
 
         <img src="./images/lang.png" alt="icon" /> Language
 
       </li>
-  <li>Javascript</li>
+     <li class="li">Javascript</li>
 
-      <li>Ruby</li>
+      <li class="li">Ruby</li>
 
-      <li>Html</li>
-
-      <li>Css</li>
-<li class="main">
+      <li class="li">Html</li>
 
+      <li class="li">Css</li>
+ 
+  
+    <li class="main" >
         <img src="./images/frame.png" alt="icon" /> Frameworks
 
       </li>
-    <li>Bootstrap</li>
+    <li class="li">Bootstrap</li>
 
-      <li>Ruby on Rails</li>
+      <li class="li">Ruby on Rails</li>
 
-      <li>RSPec</li>
+      <li class="li">RSPec</li>
 
-      <li>CapyBara</li>
+      <li class="li">CapyBara</li>
 
-      <li>Selenium</li>
- <li class="main">
+      <li class="li">Selenium</li>
+   
+  
+  
+   <li class="main">
 
         <img src="./images/skills-1.png" alt="circle-img" /> Skills
 
       </li>
-     <li>Codekit</li>
+     <li class="li">Codekit</li>
 
-      <li>Github</li>
+      <li class="li">Github</li>
 
-      <li>Codepen</li>
+      <li class="li">Codepen</li>
 
-      <li>Github</li>
+      <li class="li">Github</li>
 
-      <li>Terminal</li>
-
-  </ul>
-  </div>
+      <li class="li">Terminal</li>
+ 
+  
+ </ul>
+</div>
 
   </section>
+
+  <section class="form-section">
+    <article>I'm always interested
+         in hearing about new projects,
+         so if you'd like to chat 
+         please get in touch.
+    </article>
+         <div class="form-input">
+            <form action="https://formspree.io/f/xnqwbvpv" method="post">
+               <label for="name" class="ful-name">Full name</label>
+               <input id="name" type="text" class="full-name" maxlength="30"><br>
```

Comment:
- [ ] Kindly add the "required" attribute to activate the validation on submit for all input tags👍🏿